src/test/create-user.test.ts

```diff
@@ -1,38 +1,83 @@
 import axios from 'axios';
 import { expect } from 'chai';
+import { passwordHashing } from '../resolvers';
 import { AppDataSource } from '../data-source';
 import { User } from '../User';
-import { EXPECTED_USER, CREATE_USER_MUTATION, TEST_URL } from './test-constants';
+import { EXPECTED_USER, CREATE_USER_MUTATION, TEST_URL, headers } from './test-constants.utils';
+
+before(async () => {
+  await createFirstUser();
+});
 
 describe('User Tests', () => {
-  describe('createUser mutation', () => {
+  describe('createUser mutation Tests', () => {
     it('Should create a new user', async () => {
-      const response = await axios.post(TEST_URL, {
-        query: CREATE_USER_MUTATION,
-        variables: { data: EXPECTED_USER },
-      });
+      const response = await axios.post(
```

Comment:
Cheguei a comentar isso já, mas vc poderia organizar os testes mais ou menos assimm pra já ir se acostumando com os projetos:

- Um arquivo para cada query/mutation
- Um describe só
- Os hooks (before, beforeEach, after, afterEach) dentro do describe, na ordem mais intuititva
- Um `it` para cada caso de teste daquela query/mutation